lib/browser/browser.ts

```diff
@@ -210,48 +208,55 @@ class OktaAuthBrowser extends OktaAuthBase implements OktaAuth, SignoutAPI {
     this.authStateManager = new AuthStateManager(this);
   }
 
+  // TODO: deprecate in 5.0
   signIn(opts) {
-    const loginWithCredential = (opts) => {
-      opts = clone(opts || {});
-      const _postToTransaction = (options?) => {
-        delete opts.sendFingerprint;
-        return postToTransaction(this, '/api/v1/authn', opts, options);
-      };
-      if (!opts.sendFingerprint) {
-        return _postToTransaction();
-      }
-      return this.fingerprint()
-      .then(function(fingerprint) {
-        return _postToTransaction({
-          headers: {
-            'X-Device-Fingerprint': fingerprint
-          }
-        });
-      });
-    };
+    deprecate('This method will be deprecated in v5.0, please use signInWithCredentials() instead.');
```

Comment:
Is there something that makes this message only appear for local developers? We would not want to add console messages that would be seen by endusers. If so it could be considered a breaking change.

Reply:
Good point, we can protect this message by `feature.isLocalhost()`